aiohttp_retry/__init__.py

```diff
@@ -73,11 +74,21 @@ def _exponential_timeout(self) -> float:
     def _check_code(self, code: int) -> bool:
         return 500 <= code <= 599 or code in self._retry_options.statuses
 
+    def _request_coro(self) -> Any:
```

Comment:
I would rather call this function as "make_request" and make it async:
```python
async def _make_request(self) -> ClientResponse:
    return await self._request(...)
```

Reply:
Probably, there is no need to create a standalone function. I think that it look quite pretty:
```python
    async def _do_request(self) -> ClientResponse:
        try:
            self._current_attempt += 1
            self._logger.debug("Attempt {} out of {}".format(self._current_attempt, self._retry_options.attempts))
            response: ClientResponse = await self._request(
                self._url,
                **self._kwargs,
                trace_request_ctx={
                    'current_attempt': self._current_attempt,
                    **self._trace_request_ctx,
                }
            )
            code = response.status
```